libs/filter_lib/src/example_filter.rs

```diff
@@ -147,16 +147,27 @@ mod tests {
         let mut envoy_prop = HashMap::new();
         envoy_prop.insert("WORKLOAD_NAME".to_string(), "HI".to_string());
         let mut my_filter = Filter::new_with_envoy_properties(envoy_prop);
-        let incoming_rpc = Rpc { data: 1, uid: 1, path: String::from("ok") };
+        let incoming_rpc = Rpc {
+            data: 1,
+            uid: 1,
+            path: String::from("ok"),
+        };
         my_filter.execute(&incoming_rpc);
-        assert!(my_filter.filter_state.len()==2);
+        assert!(my_filter.filter_state.len() == 2);
 
         let state_ptr = my_filter.filter_state.get_mut("count").unwrap();
         let count_ptr = state_ptr.udf_count.as_mut().unwrap();
         count_ptr.execute();
         count_ptr.execute();
         count_ptr.execute();
-        assert!(my_filter.filter_state.get("count").unwrap().udf_count.unwrap().counter==3);
+        let udf_counter = my_filter
+            .filter_state
+            .get("count")
+            .unwrap()
+            .udf_count
+            .unwrap()
+            .counter;
+        assert!(udf_counter == 4, "Counter {} was not 4", udf_counter);
```

Comment:
This test was failing. It was expecting a value of 3 but the result is 4. Do not really understand what is expected here. As far as I can tell, the counter is incremented four times. (3x here and once in the `filter.execute` function at line 130)

Reply:
By calling my_filter.execute on line 155, plus the other 3 executes, it should be 4.  This is fine.